### PR TITLE
[Task 1848109] remove v1 legacy configuration

### PR DESCRIPTION
**Pull request by andxu**

### CODE REVIEW

**Comment by andxu** [comment]
#ab-1848109

**Comment by andxu** [comment]
AB#1848109

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/DeployMojo.java [hunk 1]
```diff
@@ -203,9 +203,6 @@ private void deploy(IWebAppBase target, WebAppConfig config) throws AzureExecuti
         }
         try {
             Log.info(String.format(DEPLOY_START, config.getAppName()));
-            if (isStopAppDuringDeployment()) {
```
Comment (Flanker32) [inline azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/DeployMojo.java:206]: This is a shared configuration in v1/v2 which support stop app before deployment, do we need to remove it?

Comment (andxu) [reply azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/DeployMojo.java]: add it back.

**Review by Flanker32 — APPROVED** [review]

**Review by shenqianjin — APPROVED** [review]

**Pull request merged**